orttraining/orttraining/python/training/ortmodule/ortmodule.py

```diff
@@ -87,7 +91,8 @@ def load_state_dict(self, state_dict: 'OrderedDict[str, Tensor]',
 
     def register_buffer(self, name: str, tensor: Optional[torch.Tensor], persistent: bool = True) -> None:
         """Override original method to delegate execution to the base module"""
-        self._original_module.register_buffer(name, tensor, persistent=persistent)
+        self._original_module.register_buffer(
```

Comment:
avoid making unnecessary code style changes. Our flake8 conf allows lines up to 128 characters.

Reply:
[peng]